.github/workflows/build_and_test.yml

```diff
@@ -71,17 +73,37 @@ jobs:
              export IPDK_RECIPE=`pwd`
              cd $IPDK_RECIPE/setup
              cmake -B build -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/deps
-             cmake --build build -j6
+             cmake --build build -j6   
 
+      - name: 'Run Stratum unit tests'
```

Comment:
Why run Stratum unit tests in the context of networking-recipe repository?

If a new PR is raised in Stratum submodule, we should run the unit tests on the open PR and/or on merge. At this point networking-recipe is still pointing to Stratum's (HEAD-1) SHA.
Networking-recipe is a follow-on separate commit and this test will only get trigger when we try to update the submodule

I think a better design/arch choice would be to do this in Stratum repo

Reply:
Interesting point.

Reply:
@ffoulkes : I think, we had a similar discussion and you mentioned Stratum tests target specific. If i remember correctly,  i too suggested to run stratum tests under stratum repo.